app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -103,7 +103,11 @@ def upload_pdf_and_attach_special_issue(form526_submission, pdf)
     end
 
     def pdf(patient_info, bpreadings, medications)
-      RapidReadyForDecision::FastTrackPdfGenerator.new(patient_info, bpreadings, medications).generate
+      assessed_data = {
+        bp_readings: bpreadings,
+        medications: medications
+      }
+      RapidReadyForDecision::FastTrackPdfGenerator.new(patient_info, assessed_data, :hypertension).generate
```

Comment:
I think it would be good to generalize this to put the disability type (it could be symbolized) instead of hypertension directly.